rules-reviewed/camel3/camel2/xml-dsl-changes.windup.xml

```diff
@@ -0,0 +1,260 @@
+                </hint>
+            </perform>
+        </rule>
+        <rule id="xml-dsl-changes-00006">
+            <when>
+                <or>
+                    <xmlfile matches="//*/b:aggregate/b:completionSize">
+                        <namespace prefix="b" uri="http://camel.apache.org/schema/blueprint"/>
+                    </xmlfile>
+                    <xmlfile matches="//*/c:aggregate/c:completionSize">
+                        <namespace prefix="c" uri="http://camel.apache.org/schema/spring"/>
+                    </xmlfile>
+                </or>
+            </when>
+            <perform>
+                <hint title="`completionSize` in aggregate was renamed" effort="1"
+                      category-id="mandatory">
+                    <message>Use
+                        Use `completionSizeExpression` instead of `completionSize`. Expressions (not the attributes) for
+                        setting correlation size/timeout were renamed.
+                    </message>
+                    <link href="https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_aggregate_eip_in_xml_dsl"
+                          title="Camel 3 - Migration Guide - Aggregate EIP changes"/>
+                    <quickfix type="REPLACE" name="camel.eip.aggregate.size.replacement">
+                        <replacement>completionSizeExpression</replacement>
+                        <search>completionSize</search>
+                    </quickfix>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="xml-dsl-changes-00007">
+            <when>
+                <or>
+                    <xmlfile matches="//*/b:aggregate/b:completionTimeout">
+                        <namespace prefix="b" uri="http://camel.apache.org/schema/blueprint"/>
+                    </xmlfile>
+                    <xmlfile matches="//*/c:aggregate/c:completionTimeout">
+                        <namespace prefix="c" uri="http://camel.apache.org/schema/spring"/>
+                    </xmlfile>
+                </or>
+            </when>
+            <perform>
+                <hint title="`completionTimeout` in aggregate was renamed" effort="1"
+                      category-id="mandatory">
+                    <message>Use
+                        Use `completionTimeoutExpression` instead of `completionTimeout`. Expressions (not the
+                        attributes) for
+                        setting correlation size/timeout were renamed.
+                    </message>
+                    <link href="https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_aggregate_eip_in_xml_dsl"
+                          title="Camel 3 - Migration Guide - Aggregate EIP changes"/>
+                    <quickfix type="REPLACE" name="camel.eip.aggregate.timeout.replacement">
+                        <replacement>completionTimeoutExpression</replacement>
+                        <search>completionTimeout</search>
```

Comment:
If we do this search all compleTmeout chunks will be replaced, so if the file already has completionTimoutExpression or completionTimeoutXXX the result will be : completionTimeoutExpressionExpression and completionTimeoutExpressionXXXX